tests/flytekit/unit/cli/pyflyte/test_run.py

```diff
@@ -223,6 +226,7 @@ def test_list_default_arguments(wf_path):
         ],
         catch_exceptions=False,
     )
+    print(result.stdout)
```

Comment:
Nit: remove this.

Reply:
no it helps in understanding the errors when it fails

Reply:
makes sense.